### ISSUE TITLE
Disable Schemas 

### ISSUE THREAD

**Issue by cheizer** [miguelgrinberg/APIFairy#70]
Hello, not sure Im asking this correctly. Is it possible to disable or hide the "SCHEMAS" section of the docs? I want to keep only the Endpoints.

**Comment by miguelgrinberg** [comment]
This project does not generate the documentation website, it just generates the OpenAPI document. This question should be addressed to the documentation renderer project that you have selected for your documentation. You may also consider taking the code from that project (they are all open source) and modify it to suit your needs.

**Comment by cheizer** [comment]
OK, thanks. I'm just using the default right now. I was hoping it was as simple as a API_FAIRY config option.

**Comment by miguelgrinberg** [comment]
No. Other than providing a very simple integration with four documentation renderers I do not want this project to get entangled with these third-party projects for which I have no control. If the default rendering does not work you can create a custom documentation endpoint and configure the renderer according to your preferences.
